database/drivers/postgres/parse.go

```diff
@@ -15,9 +15,23 @@ import (
 	"gnorm.org/gnorm/database/drivers/postgres/gnorm/tables"
 )
 
+// PG implements drivers.Driver interface for interacting with postgresql
+// database
```

Comment:
please end doc comments with a period.  I know it's dumb, but it'll fail golint :)